setup.py

```diff
@@ -31,8 +31,8 @@ def get_install_requires():
     # type: () -> List[str]
 
     install_requires = [
-        'sqlalchemy>=1.1.0', 'numpy', 'scipy', 'six', 'typing', 'cliff', 'colorlog', 'pandas',
-        'alembic'
+        'sqlalchemy>=1.1.0', 'numpy', 'scipy', 'six', 'typing<3.7.0', 'typing-extensions<3.7.0',
```

Comment:
Would you mind putting TODO comments to clarify the reason of limiting the dependency versions?

Reply:
Okay.
Added: https://github.com/pfnet/optuna/pull/434/commits/412166cbc0f4f9cf86d71ae49e1795acbaa2139c